websites/T/TeleCine/dist/metadata.json

```diff
@@ -0,0 +1,26 @@
+{
+    "author": {
+        "name": "anãø",
+        "id": "269498173259644931"
+    },
+    "service": "TeleCine",
+    "category": "videos",
+    "logo": "https://i.imgur.com/CwWONeF.png",
+  "thumbnail": "https://i.imgur.com/9TD9Gzg.jpg",
+  "version": "1.0.0",
+  "color": "#071aa6",
+  "tags": [
+    "video",
+    "stream",
+    "media"
+  ],
+  "url": [
+    "www.telecine.com.br",
+    "telecine.com.br"
```

Comment:
Website seems like it redirects to `www.*`

Reply:
https://github.com/PreMiD/Presences/pull/4892/commits/192607a8f9a7d515395483e3f7ab89edce547f38 why remove